### ISSUE TITLE
Suggestion: Link to own list of posts on website

### ISSUE THREAD

**Issue by brentajones** [microdotblog/issues#66]
I know I can see my own posts by going to micro.blog/username, but I sure can't find anywhere that links to that page. Might it be possible to add a link in the top menu or under "Account"?

**Comment by gerwitz** [comment]
This seems to have been resolved with the "Welcome" message.

**Comment by brentajones** [comment]
That's true, although once the message changes to "Up to date. There are no new posts right now," the link goes away.

**Comment by ghost** [comment]
Agree that there should be a permanent, easy-to-access profile link somewhere.

As far as _posts_ go, there is a `Posts` tab in the navigation menu but this goes specifically to the post editing listing.

I wonder if two links would make things a little easier to understand; one for your profile (micro.blog/username), another for your blog (whether hosted on Micro.blog or an external site).

**Comment by issimonwoods** [comment]
I’ve found this can sometimes be annoying. I’ll end up going into my mentions to find a link to my profile, whether on the web or iOS.
